app/cmd/root.go

```diff
@@ -75,6 +76,10 @@ func initConfig() {
 			configLoadErrorHandle(err)
 		}
 	}
+	config = getConfig()
+	if config != nil && config.Language != "" {
+		client.Language = config.Language
```

Comment:
You'd better set the language by a method. For example: `setLanguae`. It's not safe for giving a global variable.

Reply:
If I don't use global variables, there is no way to pass the values here.

Reply:
It must have a way to archive that goal.

Reply:
Maybe you have any good suggestions?